tests/attestation_plugin/enc/enc.c

```diff
@@ -271,66 +243,24 @@ static void _test_sgx_remote()
             &evidence_size),
         OE_OK);
 
-    verify_sgx_evidence(
-        &_ecdsa_report_uuid,
-        false,
-        evidence,
-        evidence_size,
-        NULL,
-        0,
-        endorsements, // reuse the endorsement generated above
-        endorsements_size,
-        test_claims,
-        TEST_CLAIMS_SIZE);
-
-    OE_TEST(oe_free_evidence(evidence) == OE_OK);
-    OE_TEST(oe_free_endorsements(endorsements) == OE_OK);
-
-    printf("testing a 65-byte custom claims that should failed\n");
-    OE_TEST_CODE(
-        oe_get_evidence(
+    {
```

Comment:
Just curious about the extra curly braces here. How does it work here?

Reply:
They define a code block to allow precise scoping of the locally defined variables (e.g. `_ecdsa_report_uuid` in this block). Without them, the variable definitions will need to be moved to the beginning of the bigger block, more prone to compiler errors or bugs.

Reply:
Thanks!